.circleci/config.yml

```diff
@@ -33,8 +35,32 @@ jobs:
           name: Verify if documentation is up-to-date
           command: npm run test:docs
 
+  deploy:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=$npmjs_token_cosmo" > ~/repo/.npmrc
+      - run:
+          name: Publish package
+          command: npm publish
+
 workflows:
   version: 2
-  build-test:
+  build-test-deploy:
     jobs:
-      - build-and-test
+      - build-test:
+          filters:
```

Comment:
Why do we need this?

Reply:
CIrcleCI reasons: https://circleci.com/docs/2.0/workflows/#git-tag-job-execution